problem3.R

```diff
@@ -0,0 +1,120 @@
+# *by* the character "|", and then we want to *trim* away all the leading and
+# trailing white spaces. The first step can be accomplished using the
+# str_split()-function in the stringr-package (this is already loaded through
+# tidyverse), but there is a delicate detail here. The "|"-character has special
+# meaning in R ("or"), so it must be *escaped*, meaning that we tell R that it
+# should be interpreted as a normal character. We do that by adding two forward
+# slashes in front of it. This function returns a list, with one element for
+# each input element. We only send one string in, and hence get only one list
+# element out (check that!). We just unlist it to get out the vector. Then we
+# apply the str_trim()-function (also in the stringr-package) to get rid of all
+# the empty space. Replace the question mark below:
+variable_names <- 
+  str_split(string = raw_file[L-1], pattern = "\\|") %>% 
+  unlist() %>% 
+  str_trim()
+
+# Read the data. One way to do this is to rewrite the data to a new .csv-file
+# with comma-separators for instance using cat() again, with the variable names
+# from the step above on the first line (see for instance paste() for collapsing
+# that vector to a single string with commas as separators).
+
+# Let us try the approach described above. It is quite transparent, but could
+# probably be done quicker. We take the elements in "raw_file" containing data,
+# replace all "|" with "," and remove all empty space. The gsub-function is
+# super for this kind of search-and-replace. Replace the question mark below.
+
+comma_separated_values <- 
+  raw_file[L+1:(length(raw_file)-L)] %>% 
+  gsub("\\|", ",", .) %>% 
+  gsub(" ", "", .)
+
+# We then just add the variable names (separated with commas) on top, and
+# cat()-the whole ting to a .csv-file in the same way as we did with the
+# variable descriptions above.
+
+comma_separated_values_with_names <- 
+  c(paste(variable_names, collapse = ","),
+    comma_separated_values)    
+
+# Replace the question mark and come up with a file name
+cat(comma_separated_values_with_names, sep = "\n", file = "galaxies.csv")
+
+# Read the file back in as a normal csv-file. The readr-package is part of
+# tidyverse, so it is already loaded.
+galaxies <- read_csv("galaxies.csv")
+
+
+# You should now have a nice, clean data frame with galaxies and their
+# characteristics in memory. As of March 2022 it should contain 796
+# observations.
+
+library(ggplot2)
+library(gridExtra)
+
+# Create two separate plots, one for values below 11 and one for values over 11
+plot_below_11 <- ggplot(data = subset(galaxies, m_b > -11), aes(x = name, y = m_b)) +
+  geom_point() +
+  labs(title = "Values Below 11")
+
+plot_above_11 <- ggplot(data = subset(galaxies, m_b <= -11), aes(x = name, y = m_b)) +
+  geom_point() +
+  labs(title = "Values Above 11")
+
+# Arrange the two plots side by side
+grid.arrange(plot_below_11, plot_above_11, ncol = 2) 
```

Comment:
You have created plots, but no interpretation. The interpretation is perhaps more important than the plot itself.